care/facility/models/file_upload.py

```diff
@@ -83,14 +83,17 @@ def save(self, *args, **kwargs):
             self.internal_name = internal_name
         return super().save(*args, **kwargs)
 
-    def signed_url(self, duration=60 * 60):
+    def signed_url(self, duration=60 * 60, mime_type=None):
         s3Client = boto3.client("s3", **cs_provider.get_client_config())
+        params = {
+            "Bucket": settings.FILE_UPLOAD_BUCKET,
+            "Key": f"{self.FileType(self.file_type).name}/{self.internal_name}",
+        }
+        if mime_type:
+            params["ContentType"] = mime_type
```

Comment:
Have you checked with a live S3 instance and tried to send an invalid payload? If so what was the response?

Reply:
When we set a content type and upload a different file (with a different mime type), the file is getting uploaded to the bucket, but it's not rendered in the iframe when fetched.

Reply:
@vigneshhari 

I tried using generate_presigned_post too, still, we can bypass the ContentType value set while creating the pre-signed URL, any suggestions to make this work? As a last option, we could proxy the file content through the backend!